cxx-async/src/lib.rs

```diff
@@ -820,6 +820,7 @@ unsafe fn unpack_exception(value: *const u8) -> CxxAsyncException {
 // * This is a low-level function called by our C++ code.
 // * `Pin<&mut Future>` is marked `#[repr(transparent)]`, so it's FFI-safe.
 // * We catch all panics inside `poll` so that they don't unwind into C++.
+#[allow(clippy::uninlined_format_args)]
```

Comment:
Let's fix them instead of adding `allow`

Reply:
done